package.json

```diff
@@ -20,7 +20,7 @@
         "@emotion/react": "^11.1.4",
         "@emotion/styled": "^11.0.0",
         "@fontsource/poppins": "^4.5.0",
-        "@prisma/client": "^2.23.0",
+        "@prisma/client": "3.0.2",
```

Comment:
This is probably causing a lock conflict in staging. Please redo these and do a yarn install again.